packages/web-api/src/WebClient.ts

```diff
@@ -964,7 +964,7 @@ function parseRetryHeaders(response: AxiosResponse): number | undefined {
  * @param logger instance of web clients logger
  */
 function warnDeprecations(method: string, logger: Logger): void {
-  const deprecatedMethods = ['workflows.'];
+  const deprecatedMethods = ['workflows.stepCompleted', 'workflows.stepFailed', 'workflows.updateStep'];
```

Comment:
📣 Somewhat related change to avoid warnings of these new methods being deprecated:

```
[WARN]  web-api:WebClient:0 workflows.featured.add is deprecated. Please check on https://docs.slack.dev/reference/methods for an alternative.
```

Reply:
Nice; did `['workflows.']` trigger this warning for any method that started with `workflows`?

Reply:
@vegeris It did and I was super confused at first, thinking that was sent from the API!

I forget when this logic was first added, but narrowing the deprecations down to exact methods seems useful going forward since this might've been warning for [adjacent](https://docs.slack.dev/reference/methods?family=workflows) methods 😳